OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -812,15 +771,16 @@ calcFiberVelocityInfo(const SimTK::State& s, FiberVelocityInfo& fvi) const
         }
 
         // Compute the other velocity-related components.
-        double dphidt = penMdl.calcPennationAngularVelocity(
+        double dphidt = getPennationModel().calcPennationAngularVelocity(
             tan(mli.pennationAngle), mli.fiberLength, dlce);
-        double dlceAT = penMdl.calcFiberVelocityAlongTendon(mli.fiberLength,
-            dlce, mli.sinPennationAngle, mli.cosPennationAngle, dphidt);
+        double dlceAT = getPennationModel().calcFiberVelocityAlongTendon(
+            mli.fiberLength, dlce, mli.sinPennationAngle, mli.cosPennationAngle,
+            dphidt);
         double dmcldt = getLengtheningSpeed(s);
         double dtl = 0;
 
         if(!get_ignore_tendon_compliance()) {
-            dtl = penMdl.calcTendonVelocity(mli.cosPennationAngle,
+            dtl = getPennationModel().calcTendonVelocity(mli.cosPennationAngle,
```

Comment:
I agree that this PR does not address all the design issues. The primary objective with this PR was to ensure that users don't encounter issues due to (1) properties being out of sync between the muscle and its helper Components, or (2) inadvertently bypassing property value checks by calling the `set_property_name()` methods instead of the value-checking `setPropertyName()` methods.